stock_move_packaging_qty/models/stock_move_line.py

```diff
@@ -9,11 +10,29 @@ class StockMoveLine(models.Model):
     product_packaging_id = fields.Many2one(
         related="move_id.product_packaging_id", readonly=True
     )
+    product_packaging_qty_reserved = fields.Float(
+        string="Reserved Pkg. Qty.",
+        help="Product packaging quantity reserved.",
+        compute="_compute_product_packaging_qty_reserved",
+        store=True,
+    )
     product_packaging_qty_done = fields.Float(
         string="Done Pkg. Qty.",
         help="Product packaging quantity done.",
     )
 
+    @api.depends("product_packaging_id", "product_uom_id", "reserved_uom_qty")
+    def _compute_product_packaging_qty_reserved(self):
+        """Get the quantity done in product packaging."""
+        for line in self:
+            if not line.product_packaging_id or not line.product_packaging_id.qty:
+                line.product_packaging_qty_reserved = False
+                continue
+            line.product_packaging_qty_reserved = float_round(
+                line.reserved_uom_qty / line.product_packaging_id.qty,
+                precision_rounding=line.product_uom_id.rounding,
+            )
```

Comment:
issue: IIUC you have to keep in mind the conversion between the line uom and the packaging uom.

For that, you can get the data from `reserved_qty` instead, [which is always in the same UoM as the product](https://github.com/odoo/odoo/blob/1d55662f424911873cb2ddcb99e3caf37f43294e/addons/stock/models/stock_move_line.py#L128-L129) (and thus the packaging).

```suggestion
            line.product_packaging_qty_reserved = float_round(
                line.reserved_qty / line.product_packaging_id.qty,
                precision_rounding=line.product_uom_id.rounding,
            )
```